src/EventHandler.cc

```diff
@@ -53,7 +54,14 @@ void EventHandler::SetFunc(FuncPtr f)
 
 void EventHandler::Call(Args* vl, bool no_remote)
 	{
-	call_count++;
+	if ( ! call_count )
+		call_count = telemetry_mgr
+		                 ->CounterFamily("zeek", "event_handler_invocations", {"name"},
```

Comment:
@timwoj this won't actually create multiple family objects. The registry of the telemetry manager checks whether the object already exists. It'll only create a new family object if there's no entry yet. Calling this once and holding on to the family object would still be faster, though, since we pay for locking a mutex and looking up the object.

Reply:
It's probably cheap enough as it's done only once per event.

I'm not sure about the passing around or putting it somewhere. Passing via Call() or the the EventHandler constructor would break the API (not detail) and also make for a sad API. I could see adding it as a static field on the EventHandler or EventRegistry class, but it would need to be wrapped in an optional, too, as non-initialized families aren't possible and that is a bit sad, too.
```

And then there's also the following - I'll push that version, but I'm happy to take guidance what else there is or would be preferred.
```
 void EventHandler::Call(Args* vl, bool no_remote)
        {
        if ( ! call_count )
               {
               static auto eh_invocations_family = telemetry_mgr->CounterFamily(
                       "zeek", "event-handler-invocations", {"name"},
                       "Number of times the given event handler was called", "1", true);

               call_count = eh_invocations_family.GetOrAdd({{"name", name}});
               }
```